sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices/api/Azure.ResourceManager.CognitiveServices.netstandard2.0.cs

```diff
@@ -1,5 +1,19 @@
 namespace Azure.ResourceManager.CognitiveServices
 {
+    public abstract partial class BaseCognitiveServicesAccountResource : Azure.ResourceManager.ArmResource
```

Comment:
Do we need a better name for this?
We have `CognitiveServicesAccountResource` and `CognitiveServicesDeletedAccountResource`. The most parts in common is just `AccountResource` which is too generic.